src/jobs/taskcluster_graph.js

```diff
@@ -84,7 +84,11 @@ export default class TaskclusterGraphJob extends Base {
     let pushdate = push.date;
 
     let level = projectConfig.level(this.config.try, repo.alias);
-    let scopes = projectConfig.scopes(this.config.try, repo.alias)
+    let scopes = projectConfig.scopes(this.config.try, repo.alias);
+
+    // All pushes should at least be able to send
+    // emails to the submitting user if desired.
+    scopes.push('route.notify.email.' + repo.user + '.*');
```

Comment:
You'll need to copy the scopes array first, or we'll have everyone emailing everyone!